gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Aug 28 17:12:58 CEST 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.1-bin.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
-zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
```

Comment:
It shouldn't have add this line. How did you perform the wrapper upgrade ?

Reply:
for me, this line is not brought by the wrapper upgrade and it is generated when i compiled this project with his previous version.

Reply:
i will rerun the commande `./gradlew wrapper --gradle-version=5.6.1`

Reply:
fixed